src/Views/batchprocessor.html

```diff
@@ -26,9 +26,20 @@ <h4 class="modal-title">StreamStats Batch Processor</h4>
     <div style="padding-top: 10px;">
         <!-- Start submit batch -->
         <div id="submitBatch" ng-if="vm.selectedBatchProcessorTabName==='submitBatch'">
-           
+            <!-- Start instructions -->
+            <div id="submitBatchInstructions"> To submit a batch: <br>

```

Comment:
No need to give a div an id if the id isn't used anywhere else

```suggestion
            <div> To submit a batch: <br>

```